og/lint/action.yml

```diff
@@ -0,0 +1,17 @@
+name: lint
+description: |
+  Run formatters/linters to OG specifications
+
+runs:
+  using: "composite"
+  steps:
+    - run: | 
+          pip install black isort docformatter flake8
+      shell: bash
+
+    - run: |
+          black .
+          isort --profile=black .
+          docformatter --recursive --in-place --wrap-summaries 88 --wrap-descriptions 88 .
+          flake8 --ignore=E203,E501,W503 .
```

Comment:
```suggestion
```
This doesn't actually format, right? It just complains if you haven't formatted things on your own.

Reply:
Yes, it only fails if modified files are not compliant with the standards, distributing the task of fixing the linting/formatting 😄

Reply:
not sure what to make of the suggestion by @heltluke - is it to be ignored or applied?